src/task2/Main.java

```diff
@@ -16,12 +16,13 @@ public static void main(String[] args) {
             for (int i = 0; i < quantityOfLines; i++) {
                 System.out.println("Enter value");
                 String value = sc.next();
-                    sumOfWords += array[i].length();
-                }
-                System.out.println("String average length is: " + sumOfWords / quantityOfLines);
-        }
-        catch (InputMismatchException er){
+                sumOfWords += array[i].length();
+            }
+            System.out.println("String average length is: " + sumOfWords / quantityOfLines);
+        } catch (InputMismatchException er) {
             System.out.println("Can`t operate this");
+        } catch (NullPointerException er2) {
+            System.out.println("Cant operate this");
```

Comment:
this error message not corresponds to the real situation: you cannot divite to 0, your error message should correspond to the reality